src/views/Dashboard.vue

```diff
@@ -178,7 +178,8 @@ export default {
       inProcessFiling: null as any,
       coaPending: false,
       coaEffectiveDate: null as string,
-      coaWarningDialog: false
+      coaWarningDialog: false,
+      tempRegNumber: sessionStorage.getItem('TEMP_REG_NUMBER')
```

Comment:
It would be better to fix this like this:
https://github.com/severinbeauvais/business-filings-ui/commit/e1c1e32d269b31ffa81b4a4d15032d1fb15f4154#diff-ac372967af9eeeb576848a16c48d6e24

Reply:
Will do, thanks!